man/man8/zpool.8

```diff
@@ -2110,6 +2159,70 @@ again.
 Starts a resilver. If an existing resilver is already running it will be
 restarted from the beginning. Any drives that were scheduled for a deferred
 resilver will be added to the new one.
```

Comment:
Add for newlines:
``` 
Starts a resilver. If an existing resilver is already running it will be
 restarted from the beginning. Any drives that were scheduled for a deferred
 resilver will be added to the new one.
+.It Xo
+.Nm
 .Cm trim
 .Op Fl p
 .Op Fl r Ar rate | Fl s
```